bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -268,6 +274,12 @@ void ReplicaImp::onMessage<ClientRequestMsg>(ClientRequestMsg *m) {
 }
 
 void ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
+  if (isSeqNumToStopAt(lastExecutedSeqNum)) {
+    LOG_INFO(GL, "Message ignored because we have to stop at super stable checkpoint");
```

Comment:
Let's make this message different from the one above. "Not sending PrePrepare message because ..." or similar.

Reply:
To clarify, I know that our current log format logs the function name. Still, I think it doesn't hurt to be more explicit.